include/ebpf_nethooks.h

```diff
@@ -6,8 +6,8 @@
 // This file contains APIs for hooks and helpers that are
 // exposed by netebpfext.sys for use by eBPF programs.
 
-// XDP hook.  We use "struct xdp_md" for cross-platform compatibility.
-typedef struct xdp_md
+// XDP_TEST hook.  We use "struct xdp_md" for cross-platform compatibility.
+typedef struct xdp_md_
```

Comment:
the xdp_md_ type shouldn't be duplicated in both repos.
```suggestion
typedef struct xdp_test_md_
```

My real preference though would be to not have xdp_test in file at all, just use what is in sample_ext.
If we need two program types in sample_ext (one that uses an ifindex and one that doesn't) that's fine, and then this might be something like "sample_program_xdp_context_" in tests\sample\ext\inc\sample_ext_helpers.h instead)